pkg/command/context.go

```diff
@@ -286,6 +288,13 @@ func createCtx(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
+func validateContextCreateFlagValues() error {
+	if contextTypeStr == "tanzu" && kubeConfig != "" {
```

Comment:
Nit: can we use `ContextTypeTanzu` instead of `"tanzu"`

Reply:
Thanks, updated and also added condition check for kubecontext and throw error if --kubecontext flag provided along with `-type tanzu` context